<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1381</width>
    <height>772</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>6</number>
     </property>
     <property name="text">
      <string>Auto Robot controller </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>524</width>
         <height>358</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1080</width>
         <height>1920</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/gg/Pictures/微信图片_20240225235052.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>524</width>
         <height>358</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1080</width>
         <height>1920</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/gg/Pictures/微信图片_20240225205435.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Red</string>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Right</string>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>48</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="tabletTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Left</string>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>36</pointsize>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>mainwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>976</x>
     <y>745</y>
    </hint>
    <hint type="destinationlabel">
     <x>783</x>
     <y>724</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>mainwindow</receiver>
   <slot>go_to_Bucket()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>729</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>617</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>go_to_Bucket()</slot>
 </slots>
</ui>
